src/v2/components/ecs-service/index.ts

```diff
@@ -21,7 +21,7 @@ export namespace EcsService {
    * - Container restarts
    * - Multiple containers
    */
-  export type PersistentStorageVolume = { name: pulumi.Input<string>; };
+  export type PersistentStorageVolume = pulumi.Input<{ name: pulumi.Input<string>; }>;
```

Comment:
Note: It might be better to not wrap types with `pulumi.Input` on the top level until they're used.

I.e. following might be better, so we leave the developer using the type to optionally wrap it:
```
type PersistentStorageVolume = { name: pulumi.Input<string>; };

type Args = { volumes: pulumi.Input<pulumi.Input<PersistentStorageVolume>[]>; };
```